syntax = "proto3";

package machine_learning.v1;

service MachineLearningService {
        rpc Ping(PingRequest) returns (PingResponse);
}

message PingRequest {
        string message = 1;
}

message PingResponse {
        string message = 1;
}